string name
---
bool success
string message


diagnostic_msgs/KeyValue[] attributes